src/store/index.ts

```diff
@@ -19,7 +19,7 @@ const state: any = {
 }
 
 const persistState = createPersistedState({
-    paths: ['user'],
+    paths: ['user','shipment','product'],
```

Comment:
We should not persist shipment and product as it may give wrong results

Reply:
Removed shipment and product from persistState.